Model/Credit_Card_Payment_Method.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Model;
+
+use \Wirecard\PaymentSdk\Config\Config;
+use \Wirecard\PaymentSdk\Config\PaymentMethodConfig;
+use \Wirecard\PaymentSdk\Transaction\Transaction;
+use \Wirecard\PaymentSdk\Transaction\CreditCardTransaction;
+
+use \OxidEsales\Eshop\Application\Model\Payment;
+use \OxidEsales\Eshop\Core\Registry;
+
+use Wirecard\Oxid\Core\Helper;
+
+/**
+ * Payment method implementation for Credit Card
+ */
+class Credit_Card_Payment_Method extends Payment_Method
+{
+    /**
+     * @inheritdoc
+     */
+    protected static $_sName = "creditcard";
+
+    /**
+     * Get the payment method's configuration
+     *
+     * @return Config
+     *
+     * @SuppressWarnings(PHPMD.Coverage)
+     */
+    public function getConfig()
+    {
+        $oPayment = oxNew(Payment::class);
+        $oPayment->load(self::getName(true));
+        $oConfig = new Config(
+            $oPayment->oxpayments__wdoxidee_apiurl->value,
+            $oPayment->oxpayments__wdoxidee_httpuser->value,
+            $oPayment->oxpayments__wdoxidee_httppass->value
+        );
+        $oPaymentMethodConfig = new PaymentMethodConfig(
+            self::getName(),
+            $oPayment->oxpayments__wdoxidee_maid->value,
+            $oPayment->oxpayments__wdoxidee_secret->value
+        );
```

Comment:
Im missing the the setting of the 3dmaid and 3dsecret

Reply:
Also for credit card you need to use CreditCardConfig

Reply:
added 3dmaid and 3dsecret and switched to CreditCardConfig.